### PR TITLE
Smooth off some rough edges

### PR DESCRIPTION
**Pull request by auscompgeek**
wpimath.trajectory.constraint summary new

DifferentialDriveKinematicsConstraint

### CODE REVIEW

**Comment by auscompgeek** [comment]
I figure none of these changes should be objectionable, but PTAL.

**Comment by virtuald** [comment]
LGTM, haven't tried it locally yet

File: robotpy_sphinx/all.py [hunk 1]
```diff
@@ -7,5 +7,4 @@ def setup(app):
     from . import pybind11_fixer
 
     app.setup_extension(pybind11_fixer.__name__)
-    app.setup_extension("sphinx_autodoc_typehints")
```
Comment (virtuald) [inline robotpy_sphinx/all.py:10]: Just kidding, why did you get rid of this?

Comment (auscompgeek) [reply robotpy_sphinx/all.py:10]: See f39b20d76ae54c4620485b1073086a9e490392a0

**Review by virtuald — APPROVED** [review]
Tried it locally. If you're happy I'm happy.

File: robotpy_sphinx/all.py (hunk 1 above)
Comment (virtuald) [reply robotpy_sphinx/all.py:10]: Does it do anything meaningfully bad though? We do have some (very few) non-pybind11 functions in various places where that could be useful.

Comment (auscompgeek) [reply robotpy_sphinx/all.py:10]: My opinion is that if we want a project to use it, then it should explicitly opt in. Apparently Sphinx 3.0 also added [a thing](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_typehints) in this area too (which works differently to the plugin).

**Pull request merged**